arm-eventhub/2015-08-01/swagger/EventHub.json

```diff
@@ -0,0 +1,1771 @@
+          "$ref": "#/definitions/NamespaceProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "Description of a Namespace resource."
+    },
+    "NamespaceProperties": {
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "description": "Provisioning state of the Namespace."
+        },
+        "status": {
+          "type": "string",
+          "description": "State of the namespace.",
+          "enum": [
+            "Unknown",
+            "Creating",
+            "Created",
+            "Activating",
+            "Enabling",
+            "Active",
+            "Disabling",
+            "Disabled",
+            "SoftDeleting",
+            "SoftDeleted",
+            "Removing",
+            "Removed",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "NamespaceState",
+            "modelAsString": false
+          }
+        },
+        "createdAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time the namespace was created."
+        },
+        "updatedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time the namespace was updated."
+        },
+        "serviceBusEndpoint": {
+          "externalDocs": {
+            "url": "http://msdn.microsoft.com/en-us/library/windowsazure/jj873988.aspx"
+          },
+          "type": "string",
+          "description": "Endpoint you can use to perform ServiceBus operations."
+        },
+        "createACSNamespace": {
+          "type": "boolean",
+          "description": "Indicates whether to create ACS namespace."
+        },
+        "enabled": {
+          "type": "boolean",
+          "description": "Specifies whether this instance is enabled."
+        }
+      },
+      "description": "Properties of the Namespace."
+    },
+    "Sku": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The resource type that this object applies to. Eg: NamespaceType",
+          "enum": [
+            "Basic",
+            "Standard",
+            "Premium"
+          ],
+          "x-ms-enum": {
+            "name": "SkuName",
+            "modelAsString": true
+          }
+        },
+        "tier": {
+          "type": "string",
+          "description": "The tier of this particular SKU",
+          "enum": [
+            "Basic",
+            "Standard",
+            "Premium"
+          ],
+          "x-ms-enum": {
+            "name": "SkuTier",
+            "modelAsString": true
+          }
+        },
+        "capacity": {
+          "format": "int32",
+          "type": "integer",
+          "description": "The messaging units or eventhub throughput units"
+        }
+      },
+      "required": [
+        "Tier"
```

Comment:
This says "**T**ier" is required, whereas the property name defined is all lowercase "**t**ier".
/cc @tbombach @kirthik  - One more rule that should be added to the linter - Ensure that the properties mentioned in the required array in the model definition actually match case sensitively with the exactly one of the properties defined in the model.